app/components/remove-prefix-postfix.js

```diff
@@ -17,7 +17,7 @@ export default Ember.Component.extend({
     return this.model.webServiceResponse.source_data.results;
   }),
   extractionFunction: Ember.computed('properties.fields.[]', function(){
-    var extractionFunction = this.model.submission.oaFields[this.get('field')].function;
+    var extractionFunction = this.model.submission.get('oaFields')[this.get('field')].function;
```

Comment:
What is the difference between `this.model` and `this.model.submission`? ( I see other controllers are using just `this.model`.  Would it be better to use `this.model.submission` for those controllers? )

Reply:
@hanbyul-here that's because for the data-format route there is an RSVP hash returned for the model, so that we can work with the web service response as well as the submission model in this route. This is only temporary, though--PR #140 will change the way this is done and it will go back to being just `this.model`. (That PR can't be merged until there is a hosted version of the data-preview service for this app to hit.